app/src/main/java/com/dcrandroid/dialog/send/SendDialog.kt

```diff
@@ -51,6 +53,8 @@ class SendDialog(val fragmentActivity: FragmentActivity, dismissListener: Dialog
 
     private var sendMax = false
```

Comment:
Add this variable to the `savedInstanceState` bundle to avoid an insufficient balance error if the `sendMax` is true .